test/auto-fix-function.js

```diff
@@ -0,0 +1,24 @@
+import testRule from 'stylelint-test-rule-tape'
+
+import declarationStrictValue, { ruleName } from '../src'
+
+const { rule } = declarationStrictValue
+
+// ignore unlisted properties
+testRule(rule, {
+  ruleName,
+  skipBasicChecks: true,
+
+  config: [
+    ["/margin/", "/padding/"], {
+    "disableFix": false,
+    "autofixFunc": function(){
+      console.log("lalalal");
+    },
+    fix: true
```

Comment:
@AndyOGo I am blocked on writing unit test. When I pass `fix` in the config test I was hoping to get the `context.fix` available at [line104](https://github.com/AndyOGo/stylelint-declaration-strict-value/pull/20/files#diff-1fdf421c05c1140f6d71444ea2b27638R104) but it comes as undefined. 

Do you have any suggestion how to proceed with this?

Reply:
@imvetri 
That is weird.

I just searched the official stylelint repo for their **autofixing** unit tests and I found this example:
https://github.com/stylelint/stylelint/blob/master/system-tests/fix/fix.test.js